lib/Runtime/Library/ScriptFunction.cpp

```diff
@@ -457,16 +457,6 @@ namespace Js
 
         ScriptContext * scriptContext = this->GetScriptContext();
 
-        if (isDefaultConstructor)
```

Comment:
Down below on line 517 of the original source is another use of isDefaultConstructor for constructing a string of the function.  Should that also be changed?

Also there is a use of IsDefaultConstructor() accessor for the same purpose in JavascriptFunctionProperty::GetValueBSTR() albeit for the BSTR api.  Again, should we change the behavior there?

If we change both those locations it looks like isDefaultConstructor becomes unused (except for one very low value assert) and we could thus remove it.